src/main/java/org/opensearchmetrics/metrics/maintainer/MaintainerMetrics.java

```diff
@@ -256,7 +254,10 @@ public List<MaintainerData> repoMaintainers(String repo) {
                             maintainerData.setName(maintainer);
                             maintainerData.setGithubLogin(githubId);
                             maintainerData.setAffiliation(affiliation);
-                            maintainersList.add(maintainerData);
+                            if(!maintainerIdSet.contains(githubId)){ // Add only unique github ids
+                                maintainerIdSet.add(githubId);
+                                maintainersList.add(maintainerData);
```

Comment:
After added to `maintainerIdSet`, where is this `maintainerIdSet` used? Still I see as `maintainersList.add(maintainerData);`.

Reply:
Line 257 to check if the id is within the set